### PR TITLE
[READY for Merge] Upgrading Terraform to 1.5 and updating Javascript to ES6

### PR DESCRIPTION
**Pull request by Crash-GHaun**
I am working on refactoring this repository. I wanted to get my build out there, I haven't had a chance to fully test it yet however I wanted to get this created for review. 

Will update as I get testing completed.

### CODE REVIEW

**Comment by Crash-GHaun** [comment]
Hello! I have completed the node and terraform updates, along with updating dependencies and moving to ES6.

File: parser-service/index.js [hunk 1]
```diff
@@ -14,42 +14,38 @@
...
- * needs to retrieve Recommender recommendations for.
- */
-app.post('/recommendation/:type', routes.applyRecommendations)
-
-/**
- * This route writes the Commit SHA and the Recommender recommendations IDs to
- * Cloud Firestore.
- */
-app.post('/ci', routes.ci)
+// Routes
```
Comment (idofl) [inline parser-service/index.js:46]: Missing original code comments.

File: parser-service/recommender.js
Comment (idofl) [inline parser-service/recommender.js:1]: Some functions are missing their original JSDoc comments

File: parser-service/sourcecontrol.js
Comment (idofl) [inline parser-service/sourcecontrol.js:1]: Some functions are missing their original JSDoc comments

**Review by idofl — APPROVED** [review]
LGTM, but please add missing code comments and JSDoc comments from the original code.

**Comment by Crash-GHaun** [comment]
Hi Ido! 

I've taken a look and I believe I've updated everything. Let me know if I am good to submit :)

**Comment by idofl** [comment]
LGTM

**Review by idofl — APPROVED** [review]

**Pull request merged**